scripts/breaking-changes.ts

```diff
@@ -0,0 +1,104 @@
+import {join, relative} from 'path';
+import {readFileSync} from 'fs';
+import {bold, red, green} from 'chalk';
+import * as ts from 'typescript';
+import * as tsutils from 'tsutils';
+
+const projectRoot = process.cwd();
+
+// Current version from the package.json. Splits it on the dash to ignore `-beta.x` suffixes.
```

Comment:
Nit to make it aligned with how we call it in the release scripts:

```suggestion
// Current version from the package.json. Splits it on the dash to ignore pre-release labels.